.github/workflows/docker-image.yml

```diff
@@ -10,6 +10,17 @@ jobs:
     steps:
       - name: ☁️ Checkout source
         uses: actions/checkout@v3
+      - name: Docker metadata action
+        id: meta_backend
+        uses: docker/metadata-action@v4.1.1
+        with:
+          images: |
+            infisical/backend
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
```

Comment:
I understand `type=ref,event=branchtype=ref,event=pr` but what will the outputs of 

`semver,pattern={{major}}.{{minor}}` and
`type=semver,pattern={{version}}` be?

Where is it getting major and minor from?

Reply:
It is pulling the git tags that already exist for releases. However, I now realize this won't really work because it is a manually run workflow and will probably just say the version is master or main.

I have another couple ideas that don't rely on the git info. I'll try them out on my own docker hub and see which is the most simple and sustainable. Using the git info just would have been convenient for creating a mechanism to manage the latest tag (I'm specifically thinking of when the most recently pushed image is not the one you want Docker offering as the default).

Reply:
I wonder if we can just use current time somehow for versioning. @reginaldbondoc do you have any thoughts on this?

Reply:
That seems reasonable if semantic versioning isn't necessary and you will never be pushing images of previous builds. If you wanted to have an image of a previous release, using current time will make it difficult to figure out which image is actually the most recent version.